samza-azure/src/main/java/org/apache/samza/ProcessorEntity.java

```diff
@@ -38,11 +39,13 @@ public ProcessorEntity(String jobModelVersion, String processorId) {
     this.partitionKey = jobModelVersion;
     this.rowKey = processorId;
     this.isLeader = false;
+    Random rand = new Random();
+    this.liveness = rand.nextInt(10000) + 2;
   }
 
   /**
    * Updates heartbeat by updating the liveness value in the table.
-   * @param value
+   * @param value Random integer value
```

Comment:
It doesn't have to be a random integer necessarily. It could be anything. What else would you suggest we do?